src/components/Messages/MessageList/livechatCall.js

```diff
@@ -0,0 +1,61 @@
+import { h } from 'preact';
+import { useState } from 'preact/compat';
+
+import I18n from '../../../i18n';
+import PhoneAccept from '../../../icons/phone.svg';
+import PhoneDecline from '../../../icons/phoneOff.svg';
+import { Avatar } from '../../Avatar';
+import { Button } from '../../Button';
+import { Screen } from '../../Screen';
+import { createClassName, getAvatarUrl } from '../../helpers';
+import { parseDate } from '../MessageTime/index';
+import styles from './styles.scss';
+
+
+const DisplayCallIframe = (rid) => (
+	<Screen.Content nopadding>
+		<div className={createClassName(styles, 'iframe')}>
+			<iframe style='height:100%' src='https://meet.jit.si/RocketChatXLKhe6QE6dyRLtTkX'{...rid} />
```

Comment:
Why is the following value hard-coded here - https://meet.jit.si/RocketChatXLKhe6QE6dyRLtTkX ? The livechat should get the jitsi server info from the core.